sccm-currentbranch/scripts/WorkFlow-SQL.ps1

```diff
@@ -0,0 +1,280 @@
+Param($DCIPAddress,$DomainFullName,$DomainAdminName,$Password,$tempurl,$sakey)
+
+add-type @"
+    using System.Net;
+    using System.Security.Cryptography.X509Certificates;
+    public class TrustAllCertsPolicy : ICertificatePolicy {
+        public bool CheckValidationResult(
+            ServicePoint srvPoint, X509Certificate certificate,
+            WebRequest request, int certificateProblem) {
+            return true;
+        }
+    }
+"@
+[System.Net.ServicePointManager]::CertificatePolicy = New-Object TrustAllCertsPolicy
+
+$Role = "SQL"
+$rolelist = "SQL Server"
+$ProvisionToolPath = "$env:windir\temp\ProvisionScript"
+if(!(Test-Path $ProvisionToolPath))
+{
+    New-Item $ProvisionToolPath -ItemType directory | Out-Null
+}
+
+$AzcopyPath = "C:\Program Files (x86)\Microsoft SDKs\Azure\AzCopy"
+
+if(!(Test-Path $AzcopyPath))
+{
+	$path = "$ProvisionToolPath\azcopy.msi"
+	if(!(Test-Path $path))
+	{
+		#Download azcopy
+		$url = "http://aka.ms/downloadazcopy"
+		Invoke-WebRequest -Uri $url -OutFile $path
+	}
+
+	#Install azcopy
+	Start-Process msiexec.exe -Wait -ArgumentList "/I $path /quiet"
+}
+
+$url = "https://cmsetoolstorage.blob.core.windows.net/work/InServicing/main.ps1"
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts